cmake/HalideGeneratorHelpers.cmake

```diff
@@ -93,8 +93,15 @@ function(add_halide_generator TARGET)
             # TODO: what do we need to do for PACKAGE_NAME PACKAGE_NAMESPACE EXPORT_FILE in this case?
         else ()
             add_executable(${TARGET} ${ARG_SOURCES})
+            target_link_libraries("${TARGET}" PRIVATE Halide::Generator ${ARG_LINK_LIBRARIES})
+            target_include_directories("${TARGET}" PRIVATE "$<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>")
```

Comment:
QQ: why is this needed? Which files do they access?

Reply:
The target_include_directories is apparently unnecessary, removing